src/pantry.js

```diff
@@ -44,9 +44,9 @@ determineCost(user, recipe) {
   return estimatedCostNeeded.join('; also ')
 }
 
-addNecessay(user, recipe){
-  checkIngredients(user, recipe)
-}
+// addNecessay(user, recipe){
```

Comment:
Is this a spelling error?

Reply:
lol well at least it's a consistent spelling error? good catch though!